src/ITensorMPS/exports.jl

```diff
@@ -0,0 +1,134 @@
+export
+  # dmrg.jl
+  dmrg,
+
+  # abstractmps.jl
+  # Macros
+  @preserve_ortho,
+  # Methods
+  AbstractMPS,
+  add,
+  common_siteind,
+  common_siteinds,
+  contract,
```

Comment:
Isn't this defined and exported by `ITensors.jl`?

Reply:
When I remove it, I get an error `UndefVarError: `contract` not defined` from line 25 of `test_abstractprojmpo.jl`. I thought maybe just importing it might be enough since ITensor exports `contract` but I guess the export is needed to also export the `contract` overloads defined by ITensorMPS?

Reply:
Maybe in that case `contract` isn't being brought into the namespace from `ITensors`. You could try adding `using ITensors: contract` at the top of that test file.

Reply:
I'll work on this and push when I've got a good solution

Reply:
See my comment. I think we should remove any exports from `ITensorMPS/exports.jl` that are already exported by `ITensors`.